functional-tests/src/test/groovy/specs/OrganizationSpecs.groovy

```diff
@@ -28,6 +28,7 @@ class OrganizationSpecs extends LoggedInSpec {
       clickCreateOrganizationButton()
       at OrganizationAddPage
     and: 'I populate all required fields to create a new organization'
+      sleep(5000)
```

Comment:
No real suggestion here, as we already brainstormed the problem together for a while.

The one thing you could do, is turn this sleep time into an environment variable, with a default of 5 seconds, so that we can control it without making code changes.

The same thing is true for the other waits, like with the getNotificationCount() above.  Which I didn't think about until now.  Not an immediate concern, but maybe something to add later.